starfish/image/_filter/bandpass.py

```diff
@@ -107,7 +114,8 @@ def run(
         verbose : bool
             if True, report on filtering progress (default = False)
         n_processes : Optional[int]
-            Number of parallel processes to devote to calculating the filter
+            Number of parallel processes to devote to calculating the filter. If None, defaults to
```

Comment:
you should just reference https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ProcessPoolExecutor, as it could change in the future (similar to how it changed for `ThreadPoolExecutor`

Reply:
I'm a bit confused. Is this relevant to current code?